src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -81,4 +81,33 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+
+    /** Stores the User in Datastore. */
+    public void storeUser(User user) {
+        Entity userEntity = new Entity("User", user.getEmail());
+        userEntity.setProperty("email", user.getEmail());
+        userEntity.setProperty("aboutMe", user.getAboutMe());
+        datastore.put(userEntity);
+    }
+
+    /**
+     * Returns the User owned by the email address, or
+     * null if no matching User was found.
+     */
+    public User getUser(String email) {
+
+        Query query = new Query("User")
+                .setFilter(new Query.FilterPredicate("email", FilterOperator.EQUAL, email));
+        PreparedQuery results = datastore.prepare(query);
+        Entity userEntity = results.asSingleEntity();
+        if(userEntity == null) {
+            return null;
+        }
+
+        String aboutMe = (String) userEntity.getProperty("aboutMe");
+        User user = new User(email, aboutMe);
```

Comment:
This works for now since you only have a small number of properties, but it is generally a good idea to use the builder pattern when it comes to classes that represent datastore entities. That is, I would recommend:

       User user = User
           .newBuilder()
           .setEmail(email)
           .setAboutMe(aboutMe)
           .build();

It's also a good idea to do it that way in that it makes it easier to populate only a subset of the fields (e.g. when making projection queries that retrieve a partial entity).